docs/settings/alert-action-settings.asciidoc

```diff
@@ -125,6 +125,11 @@ The contents of a PEM-encoded certificate file, or multiple files appended
 into a single string.  This configuration can be used for environments where
 the files cannot be made available.
 
+`xpack.actions.email.domain_allowlist`::
```

Comment:
The UI when creating a connector shows `Email address <email> is not allowed.` if the `from` is not allowed with a link to the email action type: https://www.elastic.co/guide/en/kibana/master/email-action-type.html but there is no mention of the domain allowlist on that page. Should we add a mention or a link to this allowlist entry?

Reply:
Ya, good idea, will add a link to that.